libc/src/math/generic/daddf128.cpp

```diff
@@ -0,0 +1,21 @@
+//===-- Implementation of daddf128 function -------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "src/math/daddf128.h"
+#include "include/llvm-libc-types/float128.h"
```

Comment:
Existing code transitively includes .../properties/types.h from src/math/<function>.h, so I would do the same.

```suggestion
```